solidity/test/StandardPoolConverter.js

```diff
@@ -1203,7 +1203,7 @@ describe('StandardPoolConverter', () => {
         for (const initialBalance1 of [100000, 200000, 400000, 800000]) {
             for (const initialBalance2 of [100000, 300000, 500000, 700000]) {
                 for (const conversionFeePercent of [0, 5, 10, 25, 75]) {
```

Comment:
I'd consider adding a similar case for a 100% `conversionFeePercent` (realizing that it'll slow the test, but that's life)

Reply:
Cannot; that yields division by zero.

Reply:
Wait, this can be actually a corner-case we should understand better. If the conversion fee is 100%, why wouldn't any network fee equivalent to 0%? This is something that an `if` should have solved

Reply:
I don't understand the question, but I will find out exactly what reverts and where (not related to the network-fee mechanism as far as I remember), and will let you know.

Reply:
If the conversion fee is 100%, then the returned (target) amount is obviously zero, which reverts at function `doConvert`:
```
// ensure that the trade gives something in return
require(amount != 0, "ERR_ZERO_TARGET_AMOUNT");
```

Reply:
BTW, no similar problem with the network fee (conceptual or technical), since it represents a portion of the conversion fee, not a portion of the total amount.

Reply:
This is what I wanted us to understand and you did help with that. Thanks!